packages/components/input-number/__tests__/input-number.test.tsx

```diff
@@ -301,6 +301,34 @@ describe('InputNumber.vue', () => {
     expect(wrapper.getComponent(InputNumber).emitted('focus')).toHaveLength(1)
   })
 
+  test('model-value matches input while focussed', async () => {
+    const handleChange = vi.fn()
+    const num = ref(13)
+    const wrapper = mount(() => (
+      <InputNumber v-model={num.value} onCchange={handleChange} />
```

Comment:
I believe this is a typo, innit? So I think the handler given to the component at the beginning was wrong so the test case can hold.

Reply:
@jw-foss I see, oh my... fixed it. The test will fail against the unfixed dev-branch as expected and will succeed with the fix in that PR.